tests/recordings/-pollyjs_2160168770/adapter-xhr_1803529635/Integration-XHR-Adapter_4095563232/should-be-able-to-abort-from-an-intercept_68559697/recording.har

```diff
@@ -3,27 +3,32 @@
     "_recordingName": "@pollyjs/adapter-xhr/Integration | XHR Adapter/should be able to abort from an intercept",
     "browser": {
       "name": "Chrome",
-      "version": "67.0"
+      "version": "71.0"
     },
     "creator": {
       "comment": "persister:rest",
       "name": "Polly.JS",
-      "version": "0.5.0"
+      "version": "1.4.2"
     },
     "entries": [
       {
-        "_id": "a59c75974571204bf8035e70d5ce252a",
+        "_id": "4cf42b182f2572fc502727d623cb4a3a",
         "_order": 0,
         "cache": {},
         "request": {
           "bodySize": 0,
           "cookies": [],
-          "headers": [],
-          "headersSize": 182,
+          "headers": [
+            {
+              "name": "content-type",
+              "value": "text/plain;charset=utf-8"
```

Comment:
content-type is added by nise [here](https://github.com/sinonjs/nise/blob/492703dcd4b448af684ec37a957b6144f23b4cea/lib/fake-xhr/index.js#L558-L565)